integration/common.go

```diff
@@ -324,6 +324,26 @@ func CreateCredentialApplicationJWT(credApplication credApplicationParams, crede
 	return signed.String(), nil
 }
 
+type presentationRequestParams struct {
```

Comment:
should have json tags here?

Reply:
It's used only for param replacement in tests, so not needed.